sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/ChangeFeedContextClient.java

```diff
@@ -50,6 +48,9 @@ public interface ChangeFeedContextClient {
     Flux<FeedResponse<JsonNode>> createDocumentChangeFeedQuery(CosmosAsyncContainer collectionLink,
                                                                CosmosChangeFeedRequestOptions requestOptions);
 
+    Flux<FeedResponse<ChangeFeedProcessorItem>> createDocumentChangeFeedQueryV1(CosmosAsyncContainer collectionLink,
```

Comment:
Nit: add a TODO here to come back and refactor the code. "V1" in the name is not my first choice given that we only return the new ChangeFeedProcessorItem for FF.

Reply:
We can call it `createDocumentChangeFeedQueryForFF` but keeping it separate via wire format, which we are calling V1 now. 

Also, isn't the plan that incremental will move as well to new wire format. And if that happens, we will plan to update the `handleChanges()` API to return `ChangeFeedProcessorItem` ?
If that's not the case, then I agree, we should re-name these APIs to `createDocumentChangeFeedQueryForFF`

Reply:
theoretically we don't even need a builder, we can reuse the existing one and expand it with a different handler name (which can also control which actual implementation we should call into exactly).
like handleChangesWithOperationalLogs() etc.

Reply:
the risk is then that we do need to touch on the incremental code a bit... but I think you already do that

Reply:
I am anyway touching the incremental code a little bit. So I think that's okay. Currently there is a discussion going on about what to call the Full Fidelity Change Feed Processor. I will finalize the builder after that.